src/order.cc

```diff
@@ -48,6 +67,10 @@ Range StopLimit::range() const {
   }
 }
 
+/**
+ * @param other pointer to another order

```

Comment:
Again, what is the point of this method? A: Check to see whether two orders "match", i.e. can they trade against each other?